src/main/java/frc/robot/RobotMap.java

```diff
@@ -2,8 +2,10 @@
 
 public class RobotMap {
 
-    private RobotMap() {}
+    public static final int UPPER_LIMIT_SWITCH = 1;
+    public static final int LOWER_LIMIT_SWITCH =2;
+    public static final int PISTON_FORWARD_CHANNEL = 1;
+    public static final int PISTON_REVERSE_CHANNEL = 1;
```

Comment:
```suggestion
    public static final int CORAL_ELEVATOR_PISTON_REVERSE_CHANNEL = 1;
```
Name should be specific